src/components/ebay-fake-tabs/marko-tag.json

```diff
@@ -6,6 +6,9 @@
   },
   "@html-attributes": "expression",
   "@selected-index": "number",
+  "@tab-aria-current": {
+    "enum": ["page", true]
```

Comment:
Value needs to be a string. See how `aria-current` is not being rendered properly in example:

![Screen Shot 2021-04-09 at 9 56 27 AM](https://user-images.githubusercontent.com/38065/114215126-11194400-991a-11eb-94d4-34ad0a72cee1.png)

Reply:
Also, I think `aria-current-value` would be better than `tab-aria-current`.

However, given that "current" and "true" are the only possible values, ideally we might want to use an abstraction. Something like a boolean, `tab-matches-current-url` (can't think of a good name right now) which would be `true by default and use `aria-current="page"`; if set to `false` would use `aria-current="true"`.